google/cloud/bigtable/internal/defaults_test.cc

```diff
@@ -148,6 +155,42 @@ TEST(OptionsTest, DefaultTableAdminOptions) {
   EXPECT_EQ("tableadmin.googleapis.com", options.get<EndpointOption>());
 }
 
+TEST(OptionsTest, InstanceAdminUserProjectOption) {
+  auto env = ScopedEnvironment("GOOGLE_CLOUD_CPP_USER_PROJECT", absl::nullopt);
+  auto options = DefaultInstanceAdminOptions(
```

Comment:
I would be fine with there just being one test, which calls `DefaultOptions( ... )`, instead of a test for each service... but your call.